packages/frontend/src/components/UserSettings/ProjectManagementPanel/index.tsx

```diff
@@ -70,7 +72,7 @@ const ProjectListItem: FC<{
                         })}
                     >
                         <Button
-                            icon="delete"
+                            icon="trash"
```

Comment:
why changing the icon ? I think `delete` is our standard

Reply:
x in a circle is ugly, I can change to X but trash is more obvious